src/www/build.action.mjs

```diff
@@ -20,9 +20,9 @@ import path from 'path';
 import electronConfig from './webpack_electron.mjs';
 import cordovaConfig from './webpack_cordova.mjs';
 
-import {getBuildParameters} from '../../scripts/get_build_parameters.mjs';
-import {getBuildEnvironment} from '../../scripts/get_build_environment.mjs';
-import {getWebpackBuildMode} from '../../scripts/get_webpack_build_mode.mjs';
+import {getBuildParameters} from '../build/get_build_parameters.mjs';
```

Comment:
This is fine, but I wonder if we can specify the package dependency from the root, so files can be moved without breaking the dependencies.

Maybe something like `import {getBuildParameters} from 'outline-client/build/get_build_parameters.mjs';`

Reply:
Yeah i have an idea for this, will do it next